app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -53,6 +54,10 @@ protected void onCreate(Bundle savedInstanceState) {
                     .replace(R.id.settingsContainer, new MainPreferencesFragment(), FRAGMENT_TAG)
                     .commit();
         }
+        Intent intent = getIntent();
+        if (intent.getBooleanExtra(NavListAdapter.OPEN_AUTO_DOWNLOAD_SETTINGS, false)) {
```

Comment:
Defining the constant in `NavListAdapter` feels a bit strange to me. Could you please move it to `PreferenceActivity`?

Reply:
Thanks for your feedback. I will change this :)